requirements.txt

```diff
@@ -69,14 +69,6 @@ Jinja2~=2.11.2
 jmespath~=0.10.0
 json5~=0.9.5
 jsonschema~=3.2.0
-jupyter-client~=6.1.11
-jupyter-core~=4.7.1
-jupyter-packaging~=0.9.2
-jupyter-server~=1.6.4
-jupyterlab~=3.0.14
-jupyterlab-pygments~=0.1.2
-jupyterlab-server~=2.5.0
-jupyterlab-widgets~=1.0.0
```

Comment:
Saw your message about removing this diff but I thought I would just add a reminder :)